core/models.py

```diff
@@ -1,3 +1,29 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+
+class Client(models.Model):
+    user = models.ForeignKey(User, on_delete=models.PROTECT)
+    name = models.CharField(max_length=100)
+    surname = models.CharField(max_length=100)
+    email = models.EmailField(max_length=100)
+    telephone = models.CharField(max_length=15)
```

Comment:
The phone number and the CPF could be both of 11 characters wide. Special symbols like dots and dashes are useful for visual understanding only.